packages/client/src/main.tsx

```diff
@@ -92,7 +92,6 @@ const App = () => {
             element={
               <Suspense>
                 <AppPage>
-                  <Tailwind />
```

Comment:
The reason were we're lazily importing tailwind here is because the location route breaks with it there. We need to only import it for other routes.

Reply:
fixed share menu qr

![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/fd5af6a6-4f2d-40e8-9335-248ac993abaf)